<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioIDListView</class>
 <widget class="QWidget" name="RadioIDListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Default Radio ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="defaultID"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ConfigObjectTableView" name="listView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addID">
       <property name="text">
        <string>Add Radio ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delID">
       <property name="text">
        <string>Delete Radio ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigObjectTableView</class>
   <extends>QWidget</extends>
   <header>configobjecttableview.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
